### DISCUSSION TITLE
Side mission

### DISCUSSION THREAD

**Discussion by Daek-wml** [Vdauphin/HeartsAndMinds#1545] [Question and answer] [answered]
Hello, Is there a way to spawn side missions on specific locations using the console? Thanks

**Comment by Vdauphin** [comment, marked as answer]
Hello,
No you can't without rewriting some stuff
Finding a sustainable city can be tricky: https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/side/pandemic.sqf#L27-L38
Cheers
